UI

WlodekLipski opened the pull request.

szelinskip commented on Assets/src/Controls/Detection/TouchDetector.cs:
```diff
@@ -201,6 +201,8 @@ private bool TryToResolvePlaneHold()
         {
             logger.Log("TryToResolvePlaneHold", "hold resolved");
             // at the moment ignore plane hold
```
This comment is no longer true and can be removed

szelinskip reviewed (commented):
Please squash those commits into one and rebase ui branch onto main branch, so there is only one commit in merge request, currently it is a bit messy in git log: ui -> main merge, then main -> ui merge

wlipski committed:
Polished UI

    What was done:
    * 2 new materials
    * menu for changing the material on the selected object
    * menu for setting a default material + enabling/disabling edges

    In order to enable the menu for default material or edges (on/off)
    the user should hold the point on the plane.

WlodekLipski commented:
@krecik1219 sorry for the messing up with commits.

WlodekLipski commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Add more materials (textures)](https://trello.com/c/f1nLEyF0/22-add-more-materials-textures)

WlodekLipski commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Improve UI](https://trello.com/c/l0faIIQX/21-improve-ui)
